helper/tlsdebug/inject.go

```diff
@@ -0,0 +1,31 @@
+//go:build tlsdebug
+
+package tlsdebug
+
+import (
+	"crypto/tls"
+	"os"
+	"sync"
+
+	"github.com/hashicorp/go-hclog"
+)
+
+var openKeyLogFile = sync.OnceValues(func() (*os.File, error) {
+	return os.CreateTemp("", "openbao_tls_session_key_log_")
```

Comment:
Do you want to make this an environment variable? 🤔 It seems like `pwd` may not be writable in all circumstances.

(thinking, debugging TLS issues in a k8s environment)

Reply:
This will not create a file in `pwd` but most likely at `/tmp`. When running OpenBao in K8S this will usually be writable unless the user explicitly enables a read-only root FS. And then they can still mount a volume at `/tmp` to make it writable